pkg/operator/ceph/controller/controller_utils_test.go

```diff
@@ -133,6 +133,27 @@ func TestSetEnforceHostNetwork(t *testing.T) {
 	assert.False(t, EnforceHostNetwork())
 }
 
+func TestSetRevisionHistoryLimit(t *testing.T) {
+	logger.Infof("testing default value for %v", revisionHistoryLimitSettingName)
+	opConfig := map[string]string{}
+	SetRevisionHistoryLimit(opConfig)
+	assert.Equal(t, int32(3), int32(*RevisionHistoryLimit()))
+
+	// test invalid setting
+	var value string = "foo"
+	logger.Infof("testing invalid value'%v' for %v", value, revisionHistoryLimitSettingName)
+	opConfig[revisionHistoryLimitSettingName] = value
+	SetRevisionHistoryLimit(opConfig)
+	assert.Equal(t, int32(3), int32(*RevisionHistoryLimit()))
```

Comment:
IMO incase of an invalid config we should not set the default from the Rook side, we should let kubernetes have the default value.

Reply:
@Madhu-1  wrote:
> IMO incase of an invalid config we should not set the default from the Rook side,
> we should let kubernetes have the default value.

I agree, but this code here does not look at kubernetes. It is unit test code for getting and setting  the new config variable. It does not even create a deployment or similar.

Reply:
tried to adjust the unit test to the removed default, but couldn't get it to pass locally, yet.